src/pages/admin/callouts/view/[id]/index.vue

```diff
@@ -134,6 +150,37 @@ async function confirmDeleteCallout() {
   router.push({ path: '/admin/callouts' });
 }
 
+async function endThisCallout() {
+  const now = new Date();
+  const calloutData = {
```

Comment:
hm, i get

```Argument of type '{ expires: Date; }' is not assignable to parameter of type 'AllowNull<CalloutData & CalloutFormData>'.
  Type '{ expires: Date; }' is missing the following properties from type 'AllowNull<CalloutData & CalloutFormData>': slug, title, excerpt```

and if i add `...props.callout` to have these properties, i get a 400 error with `property status should not exist`, which seems to be because `status` comes with props.callout (which i had sorted before by setting it as undefined in calloutData before feeding it to `updateCallout`)

i guess i can try removing it, but in the end it seems messier than what we had with `calloutData`. I'm sure there's a simple way out but I can't grok it easily